### PR TITLE
Node 22 test branch

### PR DESCRIPTION
**Pull request by sotojn**
This is a test branch to try and get node 22 building and running in CI

### CODE REVIEW

File: .github/workflows/publish-tag.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.19.1, 20.11.1]
+        node-version: [18.19.1, 20.11.1, 22.1.0]
```
Comment (sotojn) [inline .github/workflows/publish-tag.yml:45]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.19.1, 20.11.1]
+        node-version: [18.19.1, 20.11.1, 22.1.0]
```
Comment (sotojn) [inline .github/workflows/test.yml:35]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -55,7 +55,7 @@ jobs:
...
```
Comment (sotojn) [inline .github/workflows/test.yml:58]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -83,7 +83,7 @@ jobs:
       # opensearch is finiky, keep testing others if it fails
       fail-fast: false
       matrix:
-        node-version: [18.19.1, 20.11.1]
+        node-version: [18.19.1, 20.11.1, 22.1.0]
```
Comment (sotojn) [inline .github/workflows/test.yml:86]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -117,7 +117,7 @@ jobs:
...
```
Comment (sotojn) [inline .github/workflows/test.yml:120]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 6]
```diff
@@ -151,7 +151,7 @@ jobs:
...
```
Comment (sotojn) [inline .github/workflows/test.yml:154]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 7]
```diff
@@ -185,7 +185,7 @@ jobs:
...
```
Comment (sotojn) [inline .github/workflows/test.yml:188]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

File: .github/workflows/test.yml [hunk 8]
```diff
@@ -223,7 +223,7 @@ jobs:
...
```
Comment (sotojn) [inline .github/workflows/test.yml:226]:
```suggestion
        node-version: [18.19.1, 20.11.1, 22.2.0]
```

**Comment by godber** [comment]
We're going to leave this in draft mode until @jsnoble is done with his PR, the work here can wait.

**Pull request closed without merging**